content/fr/docs/concepts/context-propagation.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Propagation du contexte
+weight: 10
+description: Présentation du concept permettant le traçage distribué.
+default_lang_commit: 71833a5f8b84110dadf1e98604b87a900724ac33
+---
+
+La propagation du contexte permet de mettre en corrélation les
+[signaux](/docs/concepts/signals), quelle que soit leur origine. Bien qu'elle ne
+soit pas limitée au traçage, la propagation du contexte permet aux
+[traces](/docs/concepts/signals/traces) de créer des informations de causalité
+sur un système, même lorsque les services sont distribués de façon arbitraire à
+travers des limites de processus et de réseau.
```

Comment:
Ensure that update the links here to match what was done in:

- #6030

Then you'll be able to update `default_lang_commit` to e61624cde4867ae4490d2d919bd83d2b3e43c959.